src/electron/platform/android/atfa-scan-results-to-unified-results.ts

```diff
@@ -0,0 +1,119 @@
+
+export function convertAtfaScanResultsToUnifiedResults(
+    scanResults: AndroidScanResults,
+    ruleInformationProvider: RuleInformationProviderType,
+    uuidGenerator: UUIDGenerator,
+): UnifiedResult[] {
+    if (!scanResults || !scanResults.ruleResults) {
+        return [];
+    }
+
+    const unifiedResults: UnifiedResult[] = [];
+
+    for (const atfaResult of scanResults.atfaResults) {
+        if (includeBasedOnResult(atfaResult)) {
+            const viewElement: ViewHierarchyElement =
+                atfaResult['AccessibilityHierarchyCheckResult.element'];
+            if (viewElement) {
+                const ruleInformation: RuleInformation = ruleInformationProvider.getRuleInformation(
+                    atfaResult['AccessibilityHierarchyCheckResult.checkClass'],
+                );
+
+                if (ruleInformation) {
+                    unifiedResults.push(
+                        createUnifiedResult(
+                            ruleInformation,
+                            atfaResult,
+                            viewElement,
+                            uuidGenerator,
+                        ),
+                    );
+                }
+            }
+        }
+    }
+
+    return unifiedResults;
+}
+
+function createUnifiedResult(
+    ruleInformation: RuleInformation,
+    atfaResult: AccessibilityHierarchyCheckResult,
+    viewElement: ViewHierarchyElement,
+    uuidGenerator: UUIDGenerator,
+): UnifiedResult {
+    const ruleResult: RuleResultsData = {
+        axeViewId: `atfa-${viewElement['ViewHierarchyElement.id']}`,
+        ruleId: ruleInformation.ruleId,
+        status: atfaResult['AccessibilityHierarchyCheckResult.type'],
+        props: atfaResult['AccessibilityHierarchyCheckResult.metadata'],
+    };
+    return {
+        uid: uuidGenerator(),
+        ruleId: ruleInformation.ruleId,
+        status: ruleInformation.getResultStatus(ruleResult),
+        descriptors: {
+            className: viewElement?.['ViewHierarchyElement.className'],
+            boundingRectangle: convertBoundingRectangle(
+                viewElement?.['ViewHierarchyElement.boundsInScreen'],
+            ),
+            contentDescription: getRawString(
+                viewElement?.['ViewHierarchyElement.contentDescription'],
+            ),
+            text: getRawString(viewElement?.['ViewHierarchyElement.text']),
+        },
+        identifiers: {
+            identifier: viewElement?.['ViewHierarchyElement.accessibilityClassName'],
+            conciseName: viewElement?.['ViewHierarchyElement.className'],
+        },
+        resolution: ruleInformation.getUnifiedResolution(ruleResult),
+    };
+}
+
+function getRawString(spannableString?: SpannableString): string | null {
+    if (spannableString) {
+        return spannableString['SpannableString.rawString'] ?? null;
+    }
+    return null;
+}
+
+function includeBasedOnResult(atfaResult: AccessibilityHierarchyCheckResult): boolean {
```

Comment:
```suggestion
function includeBasedOnResultType(atfaResult: AccessibilityHierarchyCheckResult): boolean {
```
Naming this `includeBasedOnResultType` seems to better fit what the function does

Reply:
Done